project/.github/ISSUE_TEMPLATE/Bug.md

```diff
@@ -1,6 +1,10 @@
 ---
 name: 🐞 Bug Report
 about: Something is broken? 🔨
+title: ''
```

Comment:
what is the point of title if it is empty?

Reply:
Its generated by the github ui in this way. You . can add a default title like:
`[BUG] `, but imo its to verbose when we have the bug label

Reply:
You mean it breaks when it is missing?

Reply:
Obviously not since we already use it that way, I'm dumb… so what's the point?

Reply:
I agree with @greg0ire. If the option is not mandatory I would recommend to not add it if not used.